tooling/openshift_labeler/README.md

```diff
@@ -0,0 +1,56 @@
+# openshift-labeler
+Tool to label openshift nodes based on the role they play in openshift cluster and generate a tooling inventory on fly by looking at the node labels.
+
+### Requirements
+- Ansible
+- OpenShift inventory
+
+### Run
+```
+$ cd openshift-labeler
+$ ansible-playbook -vv -i <openshift-inventory> openshift_label.yml
+```
+
+### Labeling of openshift nodes
+The nodes in the openshift cluster are labeled as follows:
+
+- Master and etcd are co-located - role=master_etcd, type=pbench
+- Master                         - role=master, type=pbench
```

Comment:
Just a note that there is an official role for this being used in 3.9.

node-role.kubernetes.io/master=true

But, I understand what you're doing here.   Do we want to call it pbench-role?  Not a big deal, but I am on a semi-crusade against generic labels.

Reply:
I have changed the label to pbench-agent.